stdlib/csv.pyi

```diff
@@ -107,14 +110,14 @@ class DictReader(Generic[_T], Iterator[_DictReadMapping[_T, str]]):
 class DictWriter(Generic[_T]):
     fieldnames: Collection[_T]
     restval: Any | None
-    extrasaction: str
+    extrasaction: Literal["raise", "ignore"]
```

Comment:
In theory, this could be spelled in any case, but I'm fine with just allowing lower case for now.